src/acceptance/assets/file/policy/recurringschedule_dom.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
Can we substitute something other than dom which provides no real meaning, e.g., days

A better way might be to just have a single template for both, and put a comment in stating to add either days_of_month or days_of_week which you will do in code.